components/Basics.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import Profiles from "../components/Profiles";
+
+const Basics = props => {
+  let profiles = props.data.profiles.map((profile, i) => {
+    return <Profiles data={profile} key={i} />;
+  });
+  return (
+    <div>
+      <h1>{props.data.name}</h1>
+      <h4>
+        {props.data.label} - {props.data.location.region}
+      </h4>
+      <h4>
+        {props.data.location.address}, {props.data.location.city},{" "}
+        {props.data.location.postalCode} {props.data.location.countryCode}
+      </h4>
+      <img src={props.data.picture} alt="headshot" />
+      <p>{props.data.email}</p>
+      <p>{props.data.phone}</p>
+      <p>
+        <a href={props.data.website}>{props.data.website}</a>
```

Comment:
Minor comment, but one way to reduce redundancy could be to destructure the objects so that you don't have to continue writing "props.data", e.g. `const { name, label, location, website, email, phone } = props.data`